### ISSUE TITLE
api_dump layer does not work on Android

### ISSUE THREAD

**Issue by justuswang** [LunarG/VulkanTools#445]
api_dump layer is not able to generate /sdcard/Android/vk_apidump.txt on Android following the instructions in  https://github.com/LunarG/VulkanTools/blob/master/layersvt/api_dump_layer.md

**Comment by cnorthrop** [comment]
No doubt it broke before tests were available for it.  They were just added in February.  If you (or someone) gets it working, please also add a test script to ensure Android works so we can hook it up to CI.

**Comment by justuswang** [comment]
I have tried and found it works when using master@d6b567d8722850751dae7a2e2e80c4de85f23989 but not work when using master@474398dc667a09ab057802cbcdec51ba931bbd81.

But I haven't found any code change in api_dump layer itself between those two versions.
Except the layersvt/CMakeLists.txt was changed because the LVL was moved to submodule.
Not sure why it breaks api_dump layer from working on Android.
